test/unit/gen/certificate.go

```diff
@@ -261,3 +264,15 @@ func SetCertificateRevisionHistoryLimit(limit int32) CertificateModifier {
 		crt.Spec.RevisionHistoryLimit = &limit
 	}
 }
+
+func SetCertificateAdditionalKeyFormats(additionalKeyFormats []v1.AdditionalKeyFormat) CertificateModifier {
+	return func(crt *v1.Certificate) {
+		crt.Spec.PrivateKey.AdditionalKeyFormats = additionalKeyFormats
+	}
+}
+
+func EnableFeatureGate(featureGate featuregate.Feature) CertificateModifier {
+	return func(crt *v1.Certificate) {
+		utilfeature.DefaultMutableFeatureGate.Set(fmt.Sprintf("%s=%t", featureGate, true))
+	}
+}
```

Comment:
This doesn't have anything to do with building a `Certificate` object, so this function doesn't belong here.

Given it only looks like it is used once, we can probably do without this helper entirely.

Reply:
I've added Enable/Disable feature gate function in util feature gate